bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Target.scala

```diff
@@ -11,9 +11,6 @@ sealed trait Target {
   /** Coordinates (if known) for this target at the specified UNIX time. */
   def coords(time: Long): Option[Coordinates]
 
-  /** Horizons information for this target, if known. */
-  def horizonsInfo: Option[Target.HorizonsInfo]
```

Comment:
I guess this have some effects downstream but I'd assume you have checked them already